src/harness.ts

```diff
@@ -47,6 +47,28 @@ const EVENT_HANDLERS = [
 ];
 
 /**
+ * An internal function which finds a VNode based on `key`
+ * @param target The VNode to search
+ * @param key The key to find
+ */
+function findVNodebyKey(target: VNode, key: string | object): VNode | undefined {
```

Comment:
Ok, let's go with a warning when more than one `key` is detected but dispatch the event to the first key found.  I will take care of this in `findKey` as well.